src/css/components/_layout.scss

```diff
@@ -1,7 +1,15 @@
 .video-js {
-  display: block;
+  /* inline-block is as close as we get to the video el's display:inline */
+  display: inline-block;
+  /* Make video.js videos align top when next to video elements */
+  vertical-align: top;
   box-sizing: border-box;
 
+  /* Default to the video element width/height. This will be overridden by
+   * the source width height unless changed elsewhere. */
+  width: 300px;
+  height: 150px;
```

Comment:
Yeah, this is a hard one for me. We've drawn a hard line around the html5 video spec and there's a lot of benefits in being strict about that. The 300x150 default appears to come from somewhere outside of the html5 video spec, though. The canvas element defaults to 300x150 too. We'd still be going against what browsers have implemented, so a vjs player next to a bare video element wouldn't line up. But I don't think there's any argument against 16:9 being better for video than 2:1. Plus I've already set the default aspect ratio in this to 16:9 when in fluid-mode.

Unless any of that changes your mind I'll make 16x9 the default.